source/sync/configure/sync-settings.txt

```diff
@@ -451,8 +436,7 @@ directory of an :ref:`exported <export-app>` app:
        └── config.json
 
 For example, the following Sync configuration applies to apps using Flexible
-Sync (recommended). If you are using the older Partition-Based Sync, refer to
-:ref:`partition-based-sync-config-object`.
+Sync (recommended).
```

Comment:
I don't think we need to say "recommended" anymore since this is the only Sync type available for new apps.
```suggestion
Sync.
```